### ISSUE TITLE
Is this a normal result?

### ISSUE THREAD

**Issue by lijianglong2018** [eric-yyjau/pytorch-superpoint#20]
Thank you for your great work!
I just do the first training step "Training MagicPoint on Synthetic Shapes" and I look at the result by tensorBoard. The train-precision is 0.25 and the train-recall is 0.4. Is this a normal result?

**Comment by eric-yyjau** [comment]
Hi @lijianglong2018 
Thank you!
I looked at the tensorboard I had recently. It looks like the training can have precision (~0.3) and recall (0.45). Validation has precision (0.6) and recall (0.5).
Please note that validation does not have image augmentation. That's why the accuracy is much higher.
![Training precision recall](https://user-images.githubusercontent.com/17765629/90201045-871ada80-dd8e-11ea-8162-788e06a1605f.png)

![Validation precision recall](https://user-images.githubusercontent.com/17765629/90201017-79fdeb80-dd8e-11ea-832d-529c3483914f.png)

**Comment by lijianglong2018** [comment]
Thank you very much!

**Comment by ramdrop** [comment]
> I looked at the tensorboard I had recently. It looks like the training can have precision (~0.3) and recall (0.45). Validation has precision (0.6) and recall (0.5).
@eric-yyjau Hi, thank you for your time on this wonderful project. My question is: if the Precision/Recall is about 0.6, then I suppose the mAP would be about 0.5. But the original paper (Table 2) presents an mAP of >0.9. So is there anything wrong with the calculation method?

**Comment by leon5678** [comment]
> > I looked at the tensorboard I had recently. It looks like the training can have precision (~0.3) and recall (0.45). Validation has precision (0.6) and recall (0.5).
> > @eric-yyjau Hi, thank you for your time on this wonderful project. My question is: if the Precision/Recall is about 0.6, then I suppose the mAP would be about 0.5. But the original paper (Table 2) presents an mAP of >0.9. So is there anything wrong with the calculation method?

@ramdrop  hi, did you find the answer？

**Comment by ramdrop** [comment]
> > > I looked at the tensorboard I had recently. It looks like the training can have precision (~0.3) and recall (0.45). Validation has precision (0.6) and recall (0.5).
> > > @eric-yyjau Hi, thank you for your time on this wonderful project. My question is: if the Precision/Recall is about 0.6, then I suppose the mAP would be about 0.5. But the original paper (Table 2) presents an mAP of >0.9. So is there anything wrong with the calculation method?
> 
> @ramdrop hi, did you find the answer？

@leon5678 No clue yet.
